src/App.vue

```diff
@@ -27,11 +27,11 @@
 
     data: () => ({
       accessToken: process.env.VUE_APP_MAPBOX_TOKEN,
-      layers: [],
     }),
 
     computed: {
       ...mapState('app', [ 'appName' ]),
+      ...mapState('map', [ 'layers' ]),
```

Comment:
Shall we call this `rasterLayers` just to keep some distinction for ourselves?

Reply:
Changed.